localnet/common.sh

```diff
@@ -172,6 +173,28 @@ build-localnet() {
   echo "🤑 Starting LP API ..."
   KEYS_DIR=$KEYS_DIR ./$LOCALNET_INIT_DIR/scripts/start-lp-api.sh $BINARY_ROOT_PATH
 
+  echo "🔬 Starting Deposit Monitor API ..."
+  # On some machines (e.g. MacOS), 172.17.0.1 is not accessible from inside the container, so we need to use host.docker.internal
+  DEPOSIT_MONITOR_ENV='localnet/init/deposit-monitor/deposit-monitor.env'
+  if [[ $CI == true ]]; then
+    export CFDM_BROKER_API_URL='ws://172.17.0.1:10997'
+  else
+    export CFDM_BROKER_API_URL='ws://host.docker.internal:10997'
+  fi
+  $DOCKER_COMPOSE_CMD -f localnet/docker-compose.yml -p "chainflip-localnet" up $CF_CONTAINERS $additional_docker_compose_up_args -d >>$DEBUG_OUTPUT_DESTINATION 2>&1
+  while true; do
+    echo "🩺 Checking deposit-monitor's health ..."
+    REPLY=$(check_endpoint_health 'http://localhost:6060/health')
+    starting=$(echo $REPLY | jq .starting)
+    all_healthy=$(echo $REPLY | jq .all_processors)
+    if test "$starting" == "false" && test "$all_healthy" == "true" ; then
+      echo "💚 deposit-monitor is running!"
+      break
+    fi
```

Comment:
If we are checking the health here do we need to check it again in the bouncer code?

Reply:
Yes, if we want to be able to run the BLS test individually, it makes sense to ensure that the deposit-monitor is running. Though if you think better to remove this, sure.

Reply:
Yeah can see how it's useful, not a strong opinion, happy to leave if you think worthwhile

Reply:
I left a simpler test in the bouncer code, which does not keep retrying but simply fails. This should cover most scenarios.